### PR TITLE
Reducers and Actions are added

### PR DESCRIPTION
**Pull request by abiy006**
Tasks accomplished on this PR:

- Setup a Redux store
- Create a slice to save state and reducers

Changes requested by Code reviewer

- Reducer is added to booksSlice

### CODE REVIEW

File: src/redux/books/booksSlice.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /* eslint-disable no-param-reassign */
```
Comment (Shisui6) [inline src/redux/books/booksSlice.js]: Kindly go through your project, remove all lines that disable linter checks, and try to solve them yourself. Learning to solve linter errors will make you a better developer and help you write clean code instinctively😎

File: src/redux/books/booksSlice.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 import { createSlice } from '@reduxjs/toolkit';
-import { addBook, getBookList, removeBook } from '../api/getApiData';
+import { addBook, getBookList, removeBook } from '../../components/api/getApiData';
 
 const initialState = {
   data: [],
```
Comment (Shisui6) [inline src/redux/books/booksSlice.js]: According to the project requirements, you are supposed to define reducers for adding and removing books in your books slice. This has not yet been implemented. Please update your application to include this feature and remember to export them as actions🙏

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @abiy006, 

# Requires Changes 🍀
You have done a great job so far!👏👏
You still need to work on a few issues before moving on to the next project, but you're almost there🪲!

## Highlights🎯
- Follows the ducks pattern ✅
- Redux installed correctly ✅


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🪶
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by binhussen — APPROVED** [review]
Hi @abiy006 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- Added reducers and actions ✔️
-Well Structured Code✔️
- Used Git flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**